apis/config.py

```diff
@@ -0,0 +1,5 @@
+from pathlib import Path
+
+APP_ROOT    = Path("./core")
+MODELS_ROOT = APP_ROOT / "models" / "logreg.pkl"
+TEMP_CSV    = APP_ROOT / "diabetes.csv"
```

Comment:
Add new line here

Reply:
TEMP_CSV => DATA_CSV 
Also, why is it temp?

Reply:
it's temp as of now because use needs to place his/her data_path there; there could be multiple files as well;